### ISSUE TITLE
MinkUnet reproduction w/ much larger epoch

### ISSUE THREAD

**Issue by kimbss470** [mit-han-lab/spvnas#112]
Hi, Thank you for sharing your work!

When I've tried to train MinkUet@114GMACs model with 100 epochs (other hyperparameters & training configurations are default ones) ,
I got mIoU 65.28% on validation set and 62.95% on test set.

I just wanna know why you didn't use larger epochs even if it incurs better results.

**Comment by zhijian-liu** [comment]
A longer training schedule can sometimes lead to improved performance. However, it also takes much longer training time. In our original paper, we intend to keep the same number of training epochs for fair comparisons.

**Comment by deeplearning666** [comment]
Hello, When I download the pre-trained model, there is an error, like "Page Not Found. The page you are looking for doesn't exist or has been moved". I can not download the provided model, could you please share the provided model with me?
Thanking you! @kimbss470 @zhijian-liu
